### PR TITLE
Change IPv6 route prefix length from 96 to 64 in BGP BBR test

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)
Change IPv6 route prefix length from 96 to 64 in BGP BBR test to ensure that IPv6 routes from T0 are not blocked by route_map rules added by other features.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
The current BGP BBR uses an IPv6 route with prefix length 96 in test. However, this rule could be blocked by
a route_map added by other features. The route_map rule only allows IPv6 routes with prefix length 64 from
T0 VMs. Test failed because expected IPv6 routes are not found on DUT.

Another problem of this test is that the BGP BBR state is always restored to "enabled" in teardown. However,
the default_state of BBR is disabled in current image.

#### How did you do it?
Fixes:
* This change updated the IPv6 route prefix length from 96 to 64.
* Always restore BGP BBR state to the default_state defined in /etc/sonic/constants.py after test.
* Added checking of BGP BBR feature availability. If the feature is not supported, skip the test.

#### How did you verify/test it?
Test run the case on both physical and vs setup.

### CODE REVIEW

File: tests/bgp/test_bgp_bbr.py [hunk 1]
```diff
@@ -45,32 +46,43 @@ def prepare_bbr_config_files(duthosts, rand_one_dut_hostname):
     duthost.copy(content=bgp_bbr_config.render(BGP_BBR_STATUS='enabled'), dest='/tmp/enable_bbr.json')
 
 
+@pytest.fixture(scope='module')
+def bbr_default_state(setup):
+    return setup['bbr_default_state']
+
+
 def enable_bbr(duthost):
     logger.info('Disable BGP_BBR')
```
Comment (bingwang-ms) [inline tests/bgp/test_bgp_bbr.py:55]: Is the log info correct?

File: tests/bgp/test_bgp_bbr.py [hunk 2]
```diff
@@ -45,32 +46,43 @@ def prepare_bbr_config_files(duthosts, rand_one_dut_hostname):
...
     duthost.shell('sonic-cfggen -j /tmp/enable_bbr.json -w ')
+    time.sleep(3)
 
 
 def disable_bbr(duthost):
     logger.info('Enable BGP_BBR')
```
Comment (bingwang-ms) [inline tests/bgp/test_bgp_bbr.py:61]: Same as above. 'Disable BGP_BBR' ?

File: tests/bgp/test_bgp_bbr.py (hunk 1 above)
Comment (wangxin) [reply tests/bgp/test_bgp_bbr.py]: Good catch! Thanks!

File: tests/bgp/test_bgp_bbr.py (hunk 2 above)
Comment (wangxin) [reply tests/bgp/test_bgp_bbr.py]: Updated

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**